main.tf

```diff
@@ -10,7 +10,7 @@
 # }
 
 locals {
-  git_branch = "main"
+  git_branch = "ibcdpe-1004-airflow-ops"
```

Comment:
To swap back after merge.

My plan in the future is to allow us to stand up the entire infra stack when we want to test items. I had some issues with tearing down the VPC which prevented me from wanting to do this. Once those are ironed out we'll be able to start up independent infra for testing, then tear it down when we are done.